examples/cra/src/components/Alerts.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { useAlerts } from '@dhis2/app-runtime'
+
+export const Alerts = () => {
+    const alerts = useAlerts()
+
+    return alerts.map(alert => (
+        <div key={alert.id}>
+            {alert.message}
+            <button onClick={alert.remove}>hide</button>
```

Comment:
Each alert gets a `remove` property of type `function`. Calling this will remove the alert from the `alerts` state of the provider. So the assumption is that it is up to the alert to decide when this is needed.